### DISCUSSION TITLE
Wondering why watch("wells") is been ignored on my tests but not on the browser

### DISCUSSION THREAD

**Discussion by pablobuild** [react-hook-form/react-hook-form#9122] [Q&A]
Hi!

Wondering why watch("wells") is been ignored on my tests but not on the browser:

# test

![image](https://user-images.githubusercontent.com/104467391/193347263-50dfe1c2-f8db-4dec-9800-174d2b49a16e.png)

# browser

![image](https://user-images.githubusercontent.com/104467391/193347348-a5b0e69e-dc67-458a-9812-7c56a7cd3593.png)

My scenario:

<img width="663" alt="image" src="https://user-images.githubusercontent.com/104467391/193347727-7b5f5e16-849d-47c1-929d-c2d76413e2d1.png">

<img width="757" alt="image" src="https://user-images.githubusercontent.com/104467391/193347492-ebed214d-e5a3-428a-9a71-4bce7f47472a.png">

<img width="342" alt="image" src="https://user-images.githubusercontent.com/104467391/193347551-fb8ceed0-8612-4415-b2eb-57a9838b45b7.png">

<img width="235" alt="image" src="https://user-images.githubusercontent.com/104467391/193347601-1ab97fa9-2b8e-4df5-8f23-32718155e295.png">

**Comment by leapful** [comment]
@pablobuild Could you please share a codesandbox url ?
